d/config.yaml

```diff
@@ -1,5 +1,5 @@
 language:
-  version: "1.30"
+  version: "1.37"
```

Comment:
How can we be sure that latest is d 1.37 ? If you specify version on docker file I can make a PR on each slang release @cyrusmsk